src/plugins/index_pattern_field_editor/public/components/field_editor/form_fields/script_field.tsx

```diff
@@ -117,7 +117,7 @@ export const ScriptField = React.memo(({ existingConcreteFields = [], links }: P
             helpText={
               <FormattedMessage
                 id="indexPatternFieldEditor.editor.form.source.scriptFieldHelpText"
-                defaultMessage="Runtime fields without a script retrieve values from a field with the same name in {source}. If a field with the same name doesn’t exist, no values return when a search request includes the runtime field. {learnMoreLink}"
+                defaultMessage="Runtime fields without a script retrieve values from _source. If the field doesn't exist in _source, a search request returns no value. {learnMoreLink}"
```

Comment:
```suggestion
                defaultMessage="Runtime fields without a script retrieve values from a field with the same name in _source. If the field doesn't exist in _source, a search request returns no value. {learnMoreLink}"
```

Reply:
It's important to indicate that runtime fields without a script retrieve values from a field with the same name, and that the field resides in `_source`. I'm 👍 to the change for the second sentence and included it in the suggested change.

Reply:
++ to what Adam said.